app/views/proposals/_archive_link.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="col-sm-2">
-  <%- unless @approved_data.rows.size < @CLOSED_PROPOSAL_LIMIT %>
-    <%= link_to "View the archive", url_for(:controller => 'proposals', :action => 'archive') %>
-  <%- end %>
+  <% if @approved_data.rows.size > @closed_proposal_limit %>
```

Comment:
@jessieay the previous commit (72277b495152fe948b253cba94ca14ace815cf40) intentionally biased in favor of showing the link, even if there was nothing behind it. I can't seem to get into pivotal tracker to find the original card. I'm just noticing that this change basically just reverts the previous change.